### PR TITLE
I added my file via upload

### PR DESCRIPTION
**Pull request by Tess647**
THIS IS A PULL REQUEST TEMPLATE! This helps you complete a pull request. Please follow the instructions below. 

## Description
Tell us about your pull request!

**Please make sure your submission is in the correct directory**. In most cases, this is in submissions/YOUR-GITHUB-USERNAME/Week-Number/filename.

For example, if @codedentwickler is completing Week 1:
- I solved the Week 1 task.
- I create a new file inside submissions/codedentwickler/Week1/
- I name this file fizzbuzz.py, hence we have the fizzbuzz solution in submissions/codedentwickler/Week1/fizzbuzz.py

### CODE REVIEW

**Comment by Tess647** [comment]
I solved the week 1 task
I created a new file inside submissions/tess647/week 1 solution/
I named a file fizzbuzz.js, hence we have the fizzbuzz solution in submissions/tess647/week 1 solution/fizzbuzz.js

File: submissions/codedentwickler/Week1/Fizzbuzz.playground/Contents.swift [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-
-for number in 1...100 {
-    print(fizzbuzz(number))
-}
+//: Playground - noun: a place where people can play
+
+import Foundation
+
+func fizzbuzz(_ i: Int) -> String {
+    let result = (i % 3, i % 5)
```
Comment (codedentwickler) [inline submissions/codedentwickler/Week1/Fizzbuzz.playground/Contents.swift:6]: Something seems off as this looks like you edited someone else's solution (mine 😄 ). I will decline this PR and you resubmit it. Kindly ensure the changes made only reflects on the files in your submission folder.

File: submissions/tess647/README [hunk 2]
```diff
@@ -0,0 +1 @@
+tess647
```
Comment (codedentwickler) [inline submissions/tess647/README:1]: Kindly tell us more about yourself. Also, name of file should be `aboutme.md`

File: submissions/tess647/week 1 solution/fizzbuzz.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// Let's start by printing out all of the numbers from 1 and 20, 
+// and let's do it the quick and easy way. 
+for( var i=1; i<21; i++ ) {
+    if( (i%3) === 0 && (i%5) === 0 ) {
+        console.log( "FizzBuzz" );
+    }else if( (i%3) === 0 ) {
+        console.log( "Fizz" );
+    }else if( (i%5) === 0 ) {
```
Comment (codedentwickler) [inline submissions/tess647/week 1 solution/fizzbuzz.js:8]: Nice solution, kindly resubmit this in a new PR.

**Review by codedentwickler — COMMENTED** [review]
Nice solution but somethings seem off. Kindly resubmit your solution in another PR. Ensure you only change files in you submission folder.

File: submissions/codedentwickler/Week1/Fizzbuzz.playground/Contents.swift (hunk 1 above)
Comment (Tess647) [reply submissions/codedentwickler/Week1/Fizzbuzz.playground/Contents.swift:6]: Then how do I remove your solutions off the folder?

File: submissions/tess647/README (hunk 2 above)
Comment (Tess647) [reply submissions/tess647/README:1]: Okay... But it isn't necessary right

**Pull request closed without merging**